manifests/prow/deployments/prow-controller-manager.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: prow-controller-manager
+  name: prow-controller-manager
+  namespace: prow
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: prow-controller-manager
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: prow-controller-manager
+    spec:
+      containers:
+      - args:
+        - '--dry-run=false'
+        - '--config-path=/etc/config/config.yaml'
+        - '--github-token-path=/etc/github/token'
+        - '--github-endpoint=http://ghproxy'
+        - '--github-endpoint=https://api.github.com'
+        - '--enable-controller=plank'
+        #- '--health-port=8081'
+        image: gcr.io/k8s-prow/prow-controller-manager:v20210223-cdbc865141
+        livenessProbe:
+          failureThreshold: 3
+          tcpSocket:
+            port: 9090
+          # httpGet:
```

Comment:
the ports and healthchecks are commented on purpose? it applies to all yamls

Reply:
Yes, because the port seems to be there and is documented even in the `-help` command, but is not even listening. I left this commented because I want to do some further investigation in upstream to check if there is a bug or is intended.

Reply:
ok, fair enough!